R/get.R

```diff
@@ -367,13 +371,10 @@ content_list_by_tag <- function(src, tag) {
 #' @export
 content_list_guid_has_access <- function(content_list, guid) {
   warn_experimental("content_list_filter_by_guid")
-  filtered <- content_list %>% dplyr::filter(
-    access_type == "all" |
-      access_type == "logged_in" |
-      owner_guid == {{ guid }} |
-      purrr::map_lgl(permission, ~ {{ guid }} %in% .x$principal_guid)
-  )
-  return(filtered)
+  row_filter <- content_list$access_type %in% c("all", "logged_in") |
+    content_list$owner_guid == guid |
+    purrr::map_lgl(content_list$permission, ~ guid %in% .x$principal_guid)
+  content_list[row_filter, ]
```

Comment:
```suggestion
  rows_keep <- content_list$access_type %in% c("all", "logged_in") |
    content_list$owner_guid == guid |
    purrr::map_lgl(content_list$permission, ~ guid %in% .x$principal_guid)
  content_list[rows_keep, ]
```

This is super minor, but `row_filter` didn't totally connect with me at first what's going on here (and has the awkward semantics of where `filter == TRUE`, that means keep — not unique here, and totally what `dplyr:filter` does) . Other possibilities: `row_index` `rows_to_keep`

Also fine as is, it's not a huge problem